lib/manager/npm/extract/index.ts

```diff
@@ -213,6 +213,9 @@ export async function extractPackageFile(
       } else if (depName === 'yarn') {
         dep.datasource = npmId;
         dep.commitMessageTopic = 'Yarn';
+      } else if (depName === 'npm') {
+        dep.datasource = npmId;
+        dep.commitMessageTopic = 'npm';
       } else {
         dep.skipReason = SkipReason.UnknownVolta;
```

Comment:
You'll need a new test to get this code covered. 🤗

Reply:
I just updated the existing one to also cover the unknown case. Does that work?